### PR TITLE
Issue639

### PR DESCRIPTION
**Pull request by wasade**
Partially addresses #639, but is dependent on #631 for a CLI

### CODE REVIEW

**Comment by gregcaporaso** [comment]
:+1:

File: tests/test_table.py [hunk 1]
```diff
@@ -44,6 +45,49 @@
...
 class SupportTests(TestCase):
+
+    def test_head(self):
+        # example table is 2 x 3, so no change in contained data
+        exp = example_table
+        obs = example_table.head()
+        self.assertIsNot(obs, exp)
+        self.assertEqual(obs, exp)
+
+    def head_bounded(self):
```
Comment (jairideout) [inline tests/test_table.py]: Should this be `test_head_bounded`?

Comment (wasade) [inline tests/test_table.py]: hah, yes. good catch

**Comment by josenavas** [comment]
I think I commented this in the issue but, given the inherent sparsity of the data, it will be possible that showing the first (n, m) rows/cols of data will not fulfill the goal of the command, which was to see a few non-zero values.

Probably a parameter that, if passed, searches for the first non-zero value and shows the (n, m) table starting at such value might help? e.g. if the first non-zero value is at (1, 20) it shows (1, 20) to (1+n, 20+m)

**Comment by gregcaporaso** [comment]
@josenavas, I think that's a good suggestion, but let's make that change after this one goes in. This is lower priority for 2.1.5 (see the milestones for the blocking issues) so I don't want us to get bogged down in this today as we're going for the (maybe overly optimistic) goal of release today.

**Comment by josenavas** [comment]
That makes sense - thanks for the clarification @gregcaporaso !!

**Comment by wasade** [comment]
Agree

**Pull request merged**